### PR TITLE
bpo-1635741: Port math module to multiphase initialization

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue1635741

### CODE REVIEW

File: Modules/mathmodule.c [hunk 1]
```diff
@@ -3421,6 +3421,28 @@ math_ulp_impl(PyObject *module, double x)
     return x2 - x;
 }
 
+static int
+math_exec(PyObject *module) {
```
Comment (vstinner) [inline Modules/mathmodule.c]: PEP 7 recommands to put { on a separated line.

```suggestion
static int
math_exec(PyObject *module)
{
```

**Review by shihai1991 — APPROVED** [review]
LGTM

**Comment by shihai1991** [comment]
https://discuss.python.org/t/vote-to-promote-dong-hee-na/3794
Good Job, Dong-hee Na 👍

**Comment by corona10** [comment]
@shihai1991 Thanks a lot :)

**Pull request merged**